plugin-packs/postcss-preset-env/README.md

```diff
@@ -230,15 +230,25 @@ postcssPresetEnv({
 Any polyfills not explicitly enabled or disabled through `features` are
 determined by the [`stage`](#stage) option.
 
-### browsers
-
-The `browsers` option determines which polyfills are required based upon the
-browsers you are supporting.
+### env
 
-[PostCSS Preset Env] supports any standard [browserslist] configuration, which
+[PostCSS Preset Env] supports standard [browserslist] configuration, which
 can be a `.browserslistrc` file, a `browserslist` key in `package.json`, or
 `browserslist` environment variables.
 
+The `env` option is used by browserslist to determine the named environment
+it should use in the event that you have [multiple browserslist environments](https://github.com/browserslist/browserslist#configuring-for-different-environments) 
+configured. If not set, Browserslist will use the `production` environment.
+
+```js
+/* use the environment named `baseline`, instead of the default environment of  `production` */
+postcssPresetEnv({ env: 'baseline' })
+```
```

Comment:
```suggestion
The `env` option is used by [browserslist] to determine the named environment
that should be used when you have [multiple browserslist environments](https://github.com/browserslist/browserslist#configuring-for-different-environments) 
configured. If not set, Browserslist will use the `production` environment.

```js
/* use the environment named `development`, instead of the default environment of `production` */
postcssPresetEnv({ env: 'development' })
```
```


`baseline` is a loaded term in this context now that Google and MDN are marketing something with this name.

Reply:
_This suggestion was mangled by GitHub.
Better to apply it manually_